python/ml4ir/applications/ranking/tests/data/L1_sanity_tests/README

```diff
@@ -0,0 +1,8 @@
+- dataset1: 2 features, linearly separable, large margin class separation
```

Comment:
This is a README file and we can be a bit more verbose if we want to actually provide info. 
- Can be a markdown table. 
- With the cols Dataset path, Nb of points, Nb Features, is linearly separable, margin of separation,..
- It should also provide some information on how these data were generated (if one needs to generate more in the future).
- How is {small, large} margins of separations are defined?

Reply:
+1

Reply:
I recommended even just a snapshot of the `make_classification()` method used with the arguments.